### ISSUE TITLE
accessibility issue 6,7,8 (medium) - carousel

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#238]
The text on each images in this carousel has insufficient color contrast (all text on all of the images).  The color contrast should be at least 4.5:1.

The color contrast of the slide pickers (the white circles below the text on the image) is insufficient. The color contrast should be 3:1 for all user interface components.

The carousel “previous” and “next” buttons have and insufficient color contrast of 1.68:1.  (And sometimes that is even worse when the arrow is on top of an image). https://webaim.org/resources/contrastchecker/?fcolor=BABBBB&bcolor=EDF0F1
The expected color contrast ratio for interface components  is at least 3:1.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/bc5c483e54554aeb10b7f9f773fd275c9e38e370

Also carousel image is chosen at random, rather than starting with Dort

**Comment by yulgit1** [comment]
issue 23

The previous and next arrows on the carousel are links.  They should be buttons.  
The buttons should have functionality so as to work with the Enter key and the Spacebar key. (currently only works with the Enter key)

resolved above by removing previous next arrows, relying on circle indicators
